configuration/jms-setup-wildfly.cli

```diff
@@ -0,0 +1,9 @@
+connect
+batch
+
+/subsystem=messaging/hornetq-server=default/address-setting=jms.queue.TriggerMetricCollectionQueue:add(redelivery-delay=1000, max-delivery-attempts=300)
+/subsystem=messaging/hornetq-server=default/address-setting=jms.queue.MetricsQueue:add(max-delivery-attempts=-1)
+/subsystem=messaging/hornetq-server=default/address-setting=jms.queue.BatchLoadedQueue:add(max-delivery-attempts=-1)
+/subsystem=messaging/hornetq-server=default/address-setting=jms.queue.AllBatchesLoadedQueue:add(max-delivery-attempts=-1)
+
+run-batch
```

Comment:
@matzew they don't need any special handling, I believe,

the MetricsProcessingStarted topic is just "optimization" for cluster, so that it does not have to go to the DB so often...

and TriggerVariantMetricCollection will be triggered each time the notification is dispatched by PushSender, so it will eventually succeed and push TriggerMetricCollection event as a  result